src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const value in robot) {
+    const keys = robot[value];
+
+    if (Object.keys(newRobot).includes(keys)) {
+      return null;
+    }
```

Comment:
not bad, but same as in my comment above:
```suggestion
    if (Object.keys(newRobot).includes(key)) {
      return null;
    }
```